src/django_components/dependencies.py

```diff
@@ -551,8 +574,9 @@ def _postprocess_media_tags(
 
         if not _is_nonempty_str(maybe_url):
             raise RuntimeError(
-                f"One of entries for `Component.Media.{script_type}` media is missing "
-                f"value for attribute '{attr}'. Got:\n{tag}"
+                f"One of entries for `Component.Media.{script_type}` media is missing a "
```

Comment:
Updated this error message. This is raised if user defined an inlined scripts inside `Media.js`, e.g.

```py
class MyTable(Component):
    class Media:
        js = mark_safe("""
          <script>
            console.log('helo!')
          </script>
        """)
```

The thing is that we allow to pass `SafeString` to `Media.js`. And in such case we assume that we're given a string of the `<script>` tag.

But the `<script>` tag MUST contain the `src` attribute. So we ask user to move the content into a separate file and to only link to that file with this `<script>` tag.

So the result would be something like this:

```js
// hello.js
console.log('helo!')
```

```py
class MyTable(Component):
    class Media:
        js = mark_safe("""
          <script src="static/hello.js"></script>
        """)
```